### DISCUSSION TITLE
How to switch dynamic RTL/LTR on  Kendo UI for Angular without reload page?

### DISCUSSION THREAD

**Discussion by Omnia27** [abpframework/abp#7159] [Q&A]
I've tried this in each module  but is not working
when reload page is working .
please provide me with the way of how to do it properly
![provider](https://user-images.githubusercontent.com/37214596/104286660-020f2b00-54be-11eb-9552-4b9043288522.PNG)

**Comment by bnymncoskuner** [comment]
Hello,

This seems like a question for Kendo UI. It has nothing to do with ABP. We don't have a token called `RTL`.
